x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.js

```diff
@@ -0,0 +1,486 @@
+/*
```

Comment:
State is kind of message. For example, it contains `refreshConfig` and `refreshInterval` that contain the same information. How about removing anything that is in the redux store from state like refreshConfig and filters?

<img width="300" alt="Screen Shot 2020-06-17 at 3 46 05 PM" src="https://user-images.githubusercontent.com/373691/84954009-d657ce00-b0b1-11ea-943a-077396538458.png">